REST/src/main/resources/templates/playing.html

```diff
@@ -25,6 +23,6 @@ <h1>Best shooting frontend ever!</h1>
 <script src="/webjars/stomp-websocket/stomp.min.js"></script>
 <script type="text/javascript" src="/js/board.js"></script>
 
-<script>setInterval(refresh, 500)</script>
+<!--<script>setInterval(refresh, 500)</script>-->
```

Comment:
We don't need this refreshing anymore (I hope so)

Reply:
yup, but I commented it and forgot to remove it before commiting 🗡

Reply:
don't worry, tommorow we're going to kill this line with fire ;D